src/open_inwoner/ssd/xml.py

```diff
@@ -170,32 +170,36 @@ def get_uitkering_dict(xml_data) -> Optional[dict]:
     uitkeringsspecificatie = {
         "dossiernummer": {
             "key": "Dossiernummer",
-            "value": glom(dossier_dict, "Dossiernummer"),
+            "value": glom(dossier_dict, "Dossiernummer", default=""),
         },
         "periode": {
             "key": "Periode",
-            "value": format_date_month_name(glom(dossier_dict, "Periodenummer")),
+            "value": format_date_month_name(
+                glom(dossier_dict, "Periodenummer", default="")
+            ),
         },
         "regeling": {
             "key": "Regeling",
-            "value": glom(dossier_dict, "Regeling"),
+            "value": glom(dossier_dict, "Regeling", default=""),
         },
     }
 
     #
     # Details
     #
     details = {}
-    details_list = glom(dossier_dict, "Componenthistorie")
+    details_list = glom(dossier_dict, "Componenthistorie", default="")
```

Comment:
Not sure if this is wrong but it works: the default value is a string for a what is usually a list.

Reply:
Ugh, good catch. It works because both are iterable, but I'll change it.